assets/js/services/helpers.js

```diff
@@ -24,7 +24,10 @@ export function formatSimpleISODate(date) {
 }
 
 export function isUrlValid(url) {
-  return url.match(URL_PATTERN_MATCH) ? true : false;
+  return url.match(URL_PATTERN_MATCH) &&
+    URL_PATTERN_START.some((validStart) => url.startsWith(validStart))
+    ? true
```

Comment:
You probably don't even need lines 29-30 here.  Guessing it'll coerce to truthy?